modules/tools/vehicle_calibration/sensor_calibration/extract_data.py

```diff
@@ -466,24 +469,28 @@ def main():
         sys.exit(1)
 
     # Create directory to save the extracted data
-    output_path = args.output_path + re.sub(r'[^0-9]', '', str(datetime.now()))
+    output_relative_path = re.sub(r'[^0-9]', '', str(datetime.now()))
```

Comment:
Better to call `strftime` to generate readable and program friendly string.

Reply:
done